crates/cli-support/src/js/mod.rs

```diff
@@ -415,7 +429,75 @@ impl<'a> Context<'a> {
 
         self.rewrite_imports(module_name);
 
-        let mut js = if self.config.no_modules {
+        let mut js = if self.config.threads.is_some() {
+            if !self.config.no_modules {
+                // TODO: unfortunately there's bugs in bundlers like webpack, so
+                // this is only known to work with `--no-modules` for now. We
+                // should fix this with other integrations in the future! (node
+                // too!)
```

Comment:
Oh this is probably badly worded, it's definitely not active bugs moreso "this is a new world where it's totally unclear how this will all look". In that sense you're right, yeah, accessing `WebAssembly.Module` instances is one issue but another is just shipping the memory/module to a different thread and letting webpack start on the thread

Reply:
Yeah we need to think about that more.